sklearn/linear_model/_passive_aggressive.py

```diff
@@ -130,6 +130,10 @@ class PassiveAggressiveClassifier(BaseSGDClassifier):
         Number of weight updates performed during training.
         Same as ``(n_iter_ * n_samples)``.
 
+    loss_function_ : callable
+        Concrete LossFunction used to determine the loss, or
+        difference between the output of the algorithm and the target values.
```

Comment:
are you sure that there is somewhere in the code something like `self. loss_function_ = ...` to set the attribute?

Reply:
Yes in `BaseSGDClassifier` l. 496 which is the parent class

Reply:
not in the PassiveAggressiveClassifier class, but seems the attribute is inherited from the [BaseSGDClassifier class](https://github.com/scikit-learn/scikit-learn/blob/7898d6b80101caa1e8d4a3911eca242f3b86dde5/sklearn/linear_model/_stochastic_gradient.py#L496)